bower.json

```diff
@@ -14,26 +14,6 @@
     "ionic",
     "form"
   ],
-  "dependencies": {		
```

Comment:
Travis still needs these dependencies to help with bower builds within projects. While we aren't using them for our main builds, we still need them.